README.rst

```diff
@@ -3,4 +3,67 @@ Aexpect: Control your interactive applications
 
 This module provides services similar to the `pexpect` python library,
 so to speak, spawn and control interactive applications, such as `ssh`,
-`sftp` and others.
\ No newline at end of file
+`sftp` and others although it can be useful for pure streams as well.
```

Comment:
Is this the main advantage `aexpect` has over `pexpect`? Because from the first sentence it sounds almost like "this is yet another way of doing what you can already do there". It would be great to focus on what `aexpect` can do and `pexpect` right from the very start. I suggest a first sentence starting with "This project provides services similar to ... but enhanced them with ..." Note I used "project" instead of "module" since some of the remote code running (remote door with remote decorators, remote objects, and remote control files) and other additional functionality comes to mind as a definite advantage that I am not sure `pexpect` has.

Reply:
I'm not really a marketing person so I just wanted to add that it might be useful for non-bash environments as well. Improvements to this as well as the code below (remote execution) would be definitely welcome.

Reply:
I am not trying to request a sales pitch but merely to have one initial sentence that describes in short what the project is about and what separates it from other similar projects (why it exists on the first place). So IIUC you say there isn't much more that aexpect can do than pexpect? I thought the project was started because there was something Red Hat or others needed that pexpect did not already provide.